pending-documentation.md

```diff
@@ -1157,45 +1198,31 @@ rule X +Int (Y +Int Z) => (X +Int Z) +Int Y
    andBool #isVariable(Y)
 ```
 
-### `simplification` attribute (Haskell backend)
-
-The simplification attribute identifies axioms that are useful for simplifying
-configurations, without being part of the main semantics. When a rule is tagged
-as `simplification`, the Haskell backend will only apply that rule if:
+### `concrete` and `symbolic` attributes (Haskell backend)
 
--   the rule lhs _matches_ the subterm of interest, and
--   the side condition has no remainder given the current top-level predicate
-    (that is, the current top-level predicate _implies_ the side condition of
-    the rule).
+Sometimes you only want a rule to apply if some or all arguments are concrete
+(not symbolic). This is done with the `concrete` attribute. Conversely, the
+`symbolic` attribute will allow a rule to apply only when some arguments are not
+concrete. These attributes should only be given with the `simplification`
+attribute.
```

Comment:
Is it currently an error to specify both `simplification` and `concrete/symbolic`? If not, can we open an issue for that and add it to the backlog?

Reply:
Added #1470